capstone/src/main/resources/static/profile.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta>
+    <title>My Account</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+<body onload="getUser();">
+    <nav>
+        <ul>
+            <li class="active"><a href="index.html">College Activity Finder</a></li>
+            <li><a href="about.html">Find Events</a></li>
+            <li><a href="projects.html">Find Organizations</a></li>
+            <li><a href="experiences.html">My Account</a></li>
+            <li><a href="gallery.html">About Us</a></li>
+        </ul>
+    </nav>
+    <br/>
+    <div class="left">
+        <ul>
+            <li class="active"><a href="index.html">My Profile</a></li>
+            <li><a href="">Calendar of Events</a></li>
+            <li><a href="savedevents.html">Saved Events</a></li>
+            <li><a href="savedorganizations.html">Saved Organizations</a></li>
+        </ul>
+    </div>
+    <div class="profile-container">
+        <div id="auth-container"></div>
+        <div class="row">
+            <h1>My Profile</h1>
+            <div class="container">
+                <a href="images/js7.jpg"><img class="image profile" src="images/js7.jpg"/></a>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col intro">
+                <div id="firstname"></div>
+                <div id="lastname"></div>
+                <div id="email"></div>
+                <div id="user-type"></div>
+                <div id="university"></div>
+                <div id="description"></div>
+            </div>
+        </div>
+    </div>
+    <div class="profile-form">
+        <form action="save-user" method="POST">
+            <label>First Name</label>
+            <input type="text" name="firstname" placeholder="Your first name..">
+            <label>Last Name</label>
+            <input type="text" name="lastname" placeholder="Your last name..">
+            <label>Your Email</label>
+            <input type="text" name="email" id="email-form">
+            <label>Are you an individual or a club/organization?</label>
+            <select name="user-type">
+                <option value="individual">Individual</option>
+                <option value="organization">Club or Organization</option>
```

Comment:
Are you planning on creating separate entities for individuals and organizations? I know Mike mentioned it earlier. I may have missed the memo lol, but if you are, we may want to make separate forms for the two

Reply:
Yes I am making separate forms for individuals and organizations. I think the way I am going to implement this is to have a toggle selection that switches between the two forms.